glib/src/param_spec.rs

```diff
@@ -501,6 +501,48 @@ macro_rules! define_builder {
                 self
             }
 
+            /// Mark the property as readable
+            pub fn readable(mut self) -> Self {
+                self.flags |= crate::ParamFlags::READABLE;
+                self
+            }
+
+            /// Mark the property as writable
+            pub fn writable(mut self) -> Self {
+                self.flags |= crate::ParamFlags::WRITABLE;
+                self
+            }
+
```

Comment:
Those two should probably be dropped as the default is set to readwrite, in case you want to overwrite those you would go with a `T::builder(...).flags()`?

Reply:
I am talking about readable/writable here

Reply:
Yeah, having readwrite as default while having the readable/writable methods is  confusing. I would remove the readable/writable methods.

In any case, in the Properties macro I will keep using .flags(), so that it stays compatible with custom, external flags.

Reply:
Dropped those already, thanks

Reply:
You could add `readwrite()`, `read_only()` and `write_only()` that set those two flags accordingly. I think that would be a nicer API than having to call `flags()` for these cases.

Reply:
> readwrite()

That is the default, I don't see why that would be needed.  If I add a `read_only` / `write_only`, I suppose those must be called before any flags function, as those might override future flags setters?

Reply:
`readwrite()` would always set both flags (that might be unset by `read_only()` before, for example). It would just be for consistency :)

`read_only()` would set `READ` and unset `WRITE`. You can call that at any time, it doesn't overwrite any other flags.